packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -25,42 +46,61 @@
     }
 
     .nav-link {
-      background: var(--terra-paginator-nav-link-background);
-      border: var(--terra-paginator-nav-link-border);
+      background: var(--terra-paginator-nav-link-background, none);
```

Comment:
Let's go with long-hand syntax for background. [We've been burned before when we used background for a background-image](https://github.com/cerner/terra-core/issues/2426).

Reply:
Agreed. Replaced here: https://github.com/cerner/terra-core/pull/2485/commits/1812bbbad452008062754eaac49e5c2adb511e2a